CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # HEAD
 
-# [v2.3.1](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/releases/tag/v2.3.1)
+# [v2.3.2](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/releases/tag/v2.3.2)
```

Comment:
Maybe add a note that 2.3.1 is abandoned and not released

Reply:
done